mono/mini/mini-arm64.c

```diff
@@ -751,7 +757,7 @@ mono_arm_emit_aotconst (gpointer ji, guint8 *code, guint8 *code_start, int dreg,
 gboolean
 mono_arch_have_fast_tls (void)
 {
-#ifdef TARGET_IOS
+#if defined(TARGET_IOS) || defined(TARGET_OSX)
```

Comment:
Will this make both x86_64 and arm64 go through the same codepath? I'm concerned about effecting existing behavior unintentionally.

Reply:
This is in mini-arm64.c, which only gets compiled for ARM64.

Reply:
Ah yes you're right my bad.